### PR TITLE
LPD-23951

### PR DESCRIPTION
**Pull request by icgsbr**
https://liferay.atlassian.net/browse/LPD-23951

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js [hunk 1]
```diff
@@ -322,11 +322,22 @@ export default function FieldBase({
...
+					fieldsetRepeatIndexes = field.name.match(
+						FIELDSET_REPEAT_INDEX_REGEX
+					);
+				}
+				else {
+					fieldFieldsets = field.fieldName.match(FIELDSET_REGEX);
+					fieldsetRepeatIndexes = field.fieldName.match(
+						FIELDSET_REPEAT_INDEX_REGEX
+					);
+				}
```
[the remark this answers is not in the document]
Comment (icgsbr) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js:340]: That's awesome! Thanks for showing me this, I had no idea of this operator.

[the remark this answers is not in the document]
Comment (icgsbr) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js]: Done.

**Comment by icgsbr** [comment]
ci:test:sf

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:test:forms

**Comment by danvb112** [comment]
ci:test:data-engine

**Comment by icgsbr** [comment]
ci:test:relevant

**Pull request closed without merging**